src/java.desktop/macosx/classes/sun/lwawt/macosx/CPlatformWindow.java

```diff
@@ -334,7 +334,9 @@ public void initialize(Window _target, LWWindowPeer _peer, PlatformWindow _owner
 
         responder = createPlatformResponder();
         contentView.initialize(peer, responder);
-
+        if (peer != null) { // Not applicable to CWarningWindow
+            peer.setTextured(IS(TEXTURED, styleBits));
+        }
```

Comment:
This probably works fine, but maybe we can move it after "setPtr(ref.get());" below? or that ptr is not used in this new call chain?